### PR TITLE
Dropdown links

### PR DESCRIPTION
**Pull request by solowresidual**
Changed drop down links from both navbars w/ better ux on cursor hover

**From Homepage --** 
Screenshot 2020-11-26 at 12 17 53

Links to Favorites --
Screenshot 2020-11-26 at 12 18 14

**Links to My Reviews --** 
Screenshot 2020-11-26 at 12 18 22

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -16,8 +16,8 @@
       <li class="nav-item dropdown">
         <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```
Comment (johnjoubert) [inline app/views/shared/_navbar.html.erb:17]: Can we remove or change Seb's image?

File: app/views/shared/_navbar_home.html.erb [hunk 2]
```diff
@@ -14,8 +14,8 @@
       <li class="nav-item dropdown">
         <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
```
Comment (johnjoubert) [inline app/views/shared/_navbar_home.html.erb:15]: Same here... Can remove or change Seb's image?

**Review by johnjoubert — CHANGES_REQUESTED** [review]
Only change is to remove / replace the image of Seb, as that was feedback from the last project  where everyone had the same image.

Maybe while you are working in this file, you can change it quick?

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (solowresidual) [reply app/views/shared/_navbar.html.erb:17]: Yes good idea!

Comment (solowresidual) [reply app/views/shared/_navbar.html.erb]: My logo in there for now :) 

Screenshot 2020-11-26 at 12 33 27

File: app/views/shared/_navbar_home.html.erb (hunk 2 above)
Comment (solowresidual) [reply app/views/shared/_navbar_home.html.erb]: Done on both

**Review by johnjoubert — APPROVED** [review]
All good!

**Comment by b-lauren** [comment]
@solowresidual Good work! Is the podcast database logo a link back to the home page? Could be good!

**Review by b-lauren — APPROVED** [review]

**Pull request merged**